packages/tradershub/src/features/cfd/screens/CFDCompareAccounts/InstrumentsLabelHighlighted.tsx

```diff
@@ -20,10 +20,7 @@ const InstrumentsLabelHighlighted = ({ marketType, platform, shortCode }: TInstr
 
     return (
         <div
-            className={twMerge(
-                'flex flex-col rounded-24 pt-[70px] px-[15px] pb-0 lg:pt-40 lg:px-18 lg:pb-0',
-                isDemo && 'pt-16'
-            )}
+            className={twMerge('flex flex-col pt-[70px] px-[15px] lg:px-18 gap-4', isDemo && 'pt-16')}
```

Comment:
@thisyahlen-deriv Any reason Why for not having odd numbers for padding and margins?

Reply:
will change this